site/docs/v1/tech/guides/single-sign-on.adoc

```diff
@@ -698,6 +698,10 @@ Navigate to [breadcrumb]#Applications -> Your Application -> OAuth# and configur
 
 image::guides/single-sign-on/application-config-logout-behavior-docs.png[Configuring the logout behavior for an application.,width=1200,role=top-cropped]
 
+==== Limitations
```

Comment:
Can you please make this top level ( `== Limitations`) and add it to the TOC?

Reply:
I was going for a nesting under Additional Configuration > Logout Behavior > Limitations. I think it's okay to put Limitations at the end as a top level header if you'd prefer.

The page does not have ToC. Should I add one?

Reply:
I prefer limitations to be top level (lets everyone know what they are getting into).

That would be awesome if you would add a TOC.

Here's an egrep command which creates one, I typically only have the h1s and h2s in the toc:

```
egrep '^[=]+ ' site/docs/v1/tech/gides/single-sign-on.adoc |sed 's/=//' |sed 's/=/*/g'|sed 's/* /* <</'|sed 's/$/>>/'
```